eng/pipelines/runtime-official.yml

```
@@ -41,13 +41,12 @@ extends:
       # Localization build
       #
 
-      # disabled due to https://github.com/dotnet/runtime/issues/90466
-      #- ${{ if eq(variables['Build.SourceBranch'], 'refs/heads/main') }}:
-      #  - template: /eng/common/templates/job/onelocbuild.yml
-      #    parameters:
-      #      MirrorRepo: runtime
-      #      LclSource: lclFilesfromPackage
-      #      LclPackageId: 'LCL-JUNO-PROD-RUNTIME'
+      - ${{ if eq(variables['Build.SourceBranch'], 'refs/heads/main') }}:
```

This condition here seems to also be limiting the branch.

---

Seems to have been added in https://github.com/dotnet/runtime/pull/54293 cc @jkoritzinsky

---

Not entirely clear in [the docs](https://github.com/dotnet/arcade/blob/main/Documentation/OneLocBuild.md#if-youre-releasing-from-a-branch-other-than-main-including-servicing-branches), but I suspect the change should be:


```suggestion
      - ${{ if eq(variables['Build.SourceBranch'], 'refs/heads/release/8.0') }}:
```